components/automate-cli/cmd/chef-automate/status.go

```diff
@@ -75,6 +119,41 @@ func getStatus() (*api.StatusResponse, error) {
 	return res, nil
 }
 
+func runStatusSummaryCmdFunc(statusSummaryCmdFlags *StatusSummaryCmdFlags) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		return executeStatusSummary(cmd, args, statusSummaryCmdFlags)
+	}
+}
+
+func executeStatusSummary(cmd *cobra.Command, args []string, statusSummaryCmdFlags *StatusSummaryCmdFlags) error {
+	if isA2HARBFileExist() {
+		infra, err := getAutomateHAInfraDetails()
+		if err != nil {
+			return err
+		}
+		statusSummary := NewStatusSummary(infra, FeStatus{}, BeStatus{}, 10, time.Second, statusSummaryCmdFlags)
+		sshUtil := statusSummary.(*Summary).getSSHConfig()
+
+		err = statusSummary.Prepare(sshUtil)
+		if err != nil {
+			return err
+		}
+		if isManagedServicesOn() {
+			fe := statusSummary.ShowFEStatus()
```

Comment:
The 2 lines here are in if and else so you can move outside and use if !isManagedServicesOn() {Add other two lines}

Reply:
done